packages/api/db/migration/20220107134709_column_not_email_users.js

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ *  This file (20190416183353_add_column_in_fertilizer.js) is part of LiteFarm.
```

Comment:
Remove the filename from this line. It is not needed in the copyright notice, and it is often wrong due to starting with a copy of some other file.